docs/react-native/v2/features/join.mdx

```diff
@@ -3,98 +3,63 @@ title: Join Room
 nav: 3.1
 ---
 
-Joining a room is one of the most basic functionalities so we at 100ms have built an easy, developer friendly API to get you started.
+## Overview
 
-A user can interact with participants of a room only after joining the room.
+To join and interact with others in audio or video call, the user needs to join a `room`.
```

Comment:
To interact with others in audio or video call, the user needs to Join a `room`.